MREUnityRuntime/MREUnityRuntimeLib/Core/Components/AnimationComponent.cs

```diff
@@ -182,14 +188,14 @@ internal void ResetAnimation(string animationName)
             {
                 if (animation[animationName] != null)
                 {
-                    if (animation[animationName].enabled)
+                    if (animation[animationName].speed > 0)
```

Comment:
>speed > 0 [](start = 49, length = 9)

Is it possible to play an animation backward with a negative speed?

Reply:
Good catch. We don't support this in the API currently, but it might be something we expose later.